mesosturbo/main.go

```diff
@@ -12,51 +12,72 @@ import (
 
 var vmt_server_ip string
 var mesos_master_ip string
+var mesos_port string
 var target_type string
 var local_address string
 var vmt_server_port string
 var display_name string
-var marathon_master string
+var marathon_ip string
+var marathon_port string
 var opsman_username string
 var opsman_password string
+var action_api string
+var action_ip string
+var action_port string
+var slave_port string
+var username string
+var password string
+var targetid string
 
 func init() {
```

Comment:
all the vmt meta fields are set from flag? Then the command to start mesosturbo will be very very long.